### ISSUE TITLE
Join

### ISSUE THREAD

**Issue by feeskoum** [dudung/simple-website-md#1]
Can I join to develop this repository?

**Comment by dudung** [comment]
Of course. You can can start by [forking it](https://github.com/dudung/simple-website-md/fork).

**Comment by feeskoum** [comment]
> Of course. You can can start by [forking it](https://github.com/dudung/simple-website-md/fork).

Thanks. I will.

**Comment by dudung** [comment]
@feeskoum, there is new release [v0.0.1](https://github.com/dudung/simple-website-md/releases/tag/v0.0.1).
